test/io/CsvReaderTest.ooc

```diff
@@ -50,6 +50,24 @@ CsvReaderTest: class extends Fixture {
 			filename free()
 			reader free()
 		})
+		this add("non-default delimiter", func {
+			filename := Text new(c"test/io/input/semicolondelimiter.csv", 36)
```

Comment:
is the `c` prefix necessary? `t"test/io/input/semicolondelimiter.csv"` should work, no?